publicodes/docs/mecanisms.yaml

```
@@ -1,3 +1,33 @@
+applicable si:
+  chainable: oui
+  description: >-
+    Renvoit `non` si la condition est égale à `non`. Renvoit la valeur sinon.
```

coquille:

```suggestion
    Renvoie `non` si la condition est égale à `non`. Renvoie la valeur sinon.
```

idem en dessous